{{ cookiecutter.app_name }}/pyproject.toml

```diff
@@ -19,6 +19,9 @@ sources = [
 test_sources = [
     "{{ cookiecutter.test_source_dir }}",
 ]
+{% if cookiecutter.console_app %}
+console.app = "{{cookiecutter.console_app}}"
```

Comment:
```suggestion
console_app = {{ cookiecutter.console_app }}
```

Reply:
I think the if statement should stay to avoid clash between `new` and  `convert` commands, `new` command uses `pyproject_table_briefcase_app_extra_content`  attribute while `convert` command will use `console_app` attribute, if we do not use if statement we can have `console_app` statement twice in `pyproject.toml`, or we can refactor `new` command to use this attribute.

Reply:
I think you've misunderstood the suggestion I'm making here. I agree that the if statement should be retained; the suggestion is that `console.app` (period, rather than underscore) is the wrong key to use, and the value shouldn't be quoted.